tests/kas.spec.ts

```diff
@@ -120,3 +120,9 @@ test('test fail to create Kafka instance with the same name', async ({ page }) =
 
   await deleteKafkaInstance(page, testInstanceName);
 });
+
+// test_4kafka.py test_kafka_404_not_found_global
+test('404 not found Global', async ({ page }) => {
```

Comment:
this test doesn't require the setup we are doing in this file, it should go in `tests/auth.spec.ts` and possibly rename the file to something more generic.

Reply:
ok, what new name do you suggest for `auth.spce.ts` ?

Reply:
open to alternatives ... `basic.spec.ts` maybe? `generic.spec.ts` / `misc.spec.ts` ? not sure, naming is hard ...

Reply:
ok so `basic.spec.ts` moving and we can merge and spend no more time on it